app/controllers/items_controller.rb

```diff
@@ -138,18 +179,16 @@ def workflow_update
     if args.all?(&:present?)
       # the :repo parameter is optional, so fetch it based on the workflow name if blank
       params[:repo] ||= Dor::WorkflowObject.find_by_name(params[:wf_name]).definition.repo
-
       # this will raise an exception if the item doesn't have that workflow step
       Dor::WorkflowService.get_workflow_status params[:repo], *args.take(3)
-
       # update the status for the step and redirect to the workflow view page
       Dor::WorkflowService.update_workflow_status params[:repo], *args
-
       respond_to do |format|
         if params[:bulk].present?
           render :status => 200, :text => 'Updated!'
         else
-          format.any { redirect_to catalog_path(params[:id]), :notice => "Updated #{params[:process]} status to '#{params[:status]}' in #{params[:wf_name]}" }
+          msg = "Updated #{params[:process]} status to '#{params[:status]}' in #{params[:wf_name]}"
+          format.any { redirect_to catalog_path(params[:id]), notice: msg }
```

Comment:
Any functional changes here? Prefer not to make changes in this PR that aren't related to current issue.